app/controllers/development_metrics_controller.rb

```diff
@@ -54,10 +56,19 @@ def build_metrics(entity_id, entity_name)
   end
 
   def build_product_metrics(entity_id, entity_name)
+    @has_jira_project = Product.find(product.id).jira_project&.present?
+
+    return unless @has_jira_project
+
+    show_metrics?
+
     metrics = Builders::Chartkick::DevelopmentMetrics.const_get(entity_name)
                                                      .call(entity_id, metric_params[:period])
-    @defect_escape_rate = metrics[:defect_escape_rate]
-    @metric_definition = MetricDefinition.find_by(code: :defect_escape_rate)
+
+    defect_escape_rate(metrics)
+    development_cycle(metrics)
+
+    @query_metric_name = metric_params[:name]
```

Comment:
You can set this variable at the beginning of the method and use it in the `show_metrics?` method